data/helpers.d/filesystem

```diff
@@ -60,8 +61,14 @@ ynh_backup() {
     # ==============================================================================
     # Be sure the source path is not empty
     [[ -e "${SRC_PATH}" ]] || {
-        echo "Source path '${SRC_PATH}' does not exist" >&2
-        return 1
+        echo "!!! Source path '${SRC_PATH}' does not exist !!!" >&2
+        if [ $NEEDED -eq 1 ]
+        then
+                return 1
+        else
+                echo "The missing file will be replaced by a dummy one for the backup !!!" >&2
+                echo "This file is dummy... Please contact the maintainer of this app..." | tee "${SRC_PATH}" > /dev/null
```

Comment:
I suggest to remove this line. It could break the system by creating a conf file with a bad content not interpretable by the service that manage this conf file

Reply:
Indeed, forgot that...
Still, an admin should be informed that there were an error with the backup.

Reply:
Mybe we just can inform the admin that the file is missing and explain how to create a dummy file.

Reply:
Create a dummy file is what is going to save the process. So we can't avoid that otherwise this PR has no meaning.
But the question was more, how to inform an admin ?